data_ingestion/04_Ingest_data_with_EMR.ipynb

```diff
@@ -0,0 +1,237 @@
+{
```

Comment:
<p>Add space:</p><blockquote>command.You</blockquote><p> </p><p>Update </p><blockquote> For more details, please see <a href="https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-spark-s3select.html" target="_blank">documentation</a>.</blockquote><p>to:</p><blockquote>  For more details, see the documentation on <a href="https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-spark-s3select.html" target="_blank">using S3 select with Spark</a>.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1602/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>